Processing_data/Interfaces/IPrint.cs

```diff
@@ -0,0 +1,9 @@
+using Processing_data.Classes;
+
+namespace Processing_data.Interfaces
+{
+    public interface IPrint
```

Comment:
Please review the naming guidelines: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines.

> DO name interfaces with adjective phrases, or occasionally with nouns or noun phrases.
> DO prefix interface names with the letter I, to indicate that the type is an interface.

Here, it means that the` IPrinter` name fits better.